projects/product-configuration/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@spartacus/product-configuration",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^9.1.0",
+    "@angular/core": "^9.1.0",
+    "tslib": "^1.10.0",
+    "@spartacus/core": "2.1.0-dev.0",
+    "@spartacus/storefront": "2.1.0-dev.0"
+  }
+}
```

Comment:
Feel free to add `description`, `homepage`, `keywords` and `repository`.
Also we should add `license` and `publishConfig`.

A nice example for this is `projects/cds/package.json`.